packages/react/src/Pagination/Pagination.tsx

```diff
@@ -158,7 +34,6 @@ export type PageProps = {
 } & Omit<PageDataProps['props'], 'as' | 'role'>
 
 type UsePaginationPagesParameters = {
-  theme?: Record<string, unknown> // set to theme type once /src/theme.js is converted
```

Comment:
wasn't totally sure what to do here, it didn't seem like it was being used by our CSS modules, so I removed it.